css/base.css

```diff
@@ -1,3 +1,6 @@
+
+/* TODO There is so much I want to change witin the styling and look of this project. First of all I need to implement the hamburger menu. Next, I am designing from a mobile-first perspective so I want the hamburger menu and one project per section. I want to create a media query that will change for the desktop version of my portfolio. My desktop version will display 2-3 portfolios in a line. Also my hamburger menu should change into an inline nav bar. I also want to change my header to be more stylish. Since the content on my page will be portfolio projects and look fairly basic I want the header to be a place where I can showcase my design skills. My about me section is another area where I would like to showcase my CSS abilities. Also as I mentioned previously, I would like to add another section to my portfolio that is a personal resume. This will showcase my background but also my ability to create a resume online using web development skills. */
+
```

Comment:
I like the way you wrote out your TODO's in almost a story format, as if you are having a natural conversation with yourself in regards to what needs to be done, and what you would like to work towards. You've laid out the overview of what you deem as needing to be done/fixed, which will help you work on immediate action items, along with a great map that will keep your perspective as you complete your Portfolio. I will do my best to add to your already well-thought out TODO's.

Great job considering the end-design of mobile-first. As you wrote in my peer-review, hamburger menu for the nav bar for mobile-perspective makes a lot of sense. Keeping the user perspective in mind and how this needs to be laid out depending on user-perspective is key in having a successful outcome in your Portfolio, and you are already thinking that way which is setting you up for success. 

My suggestion is to list by importance so you can tackle the necessary first, which I can tell you already have in mind, but it's easy to view importance in priorities.  I know that seems obvious, and I can tell you have this mapped out already in your mind from the way you wrote out the above, but having it listed in order of importance allows you to have a flexible map of TODO's without constantly having to remap them in your head (1 less thing to have to constantly think about! :) ), and tackle your TODO's methodically, while allowing you to be flexible as you have to adapt to things that come up.  That helps me, but you know what works for you, and I think you have great points laid out!